docs/QUnit/only.md

```diff
@@ -23,9 +23,7 @@ Adds a test to exclusively run, preventing all other tests from running.
 
 ### Description
 
-Use this method to focus your test suite on a specific test. `QUnit.only` will cause any other tests in your suite to be ignored.
-
-Note that if more than one `QUnit.only` is present only the first instance will run.
+Use this method to focus your test suite on a specific set of tests. `QUnit.only` will cause any other tests in your suite to be ignored.
```

Comment:
After the change above, the first part of this could probably be updated to `...to focus your test suite on specific tests.`

Reply:
Updated.